poly-commitment/src/evaluation_proof.rs

```diff
@@ -321,8 +321,8 @@ pub struct OpeningProof<G: AffineCurve> {
 }
```

Comment:
I am not sure what happens with the structs that have an ocaml equivalent, can these be `pub(crate)` as well?